### PR TITLE
Update to new headers

### PR DESCRIPTION
**Pull request by sophistifunk**
# Description

New headers.

See [JENKINS-39762](https://issues.jenkins-ci.org/browse/JENKINS-39762).
See [JENKINS-39763](https://issues.jenkins-ci.org/browse/JENKINS-39763).

# Submitter checklist
- [X] Link to JIRA ticket in description, if appropriate.
- [X] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

**Comment by michaelneale** [comment]
The ATH run here: https://ci.blueocean.io/job/ATH-Jenkinsfile/job/master/1174/console

cc @tfennelly @scherler

**Comment by i386** [comment]
I've started putting feedback in the [Header update epic](https://issues.jenkins-ci.org/browse/JENKINS-41552).

**Comment by michaelneale** [comment]
(lint failures)

File: blueocean-dashboard/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "blueocean-dashboard",
   "version": "0.0.1",
   "scripts": {
-    "start-storybook": "start-storybook --static-dir ./target,./node_modules/@jenkins-cd/design-language/dist/assets/",
+    "start-storybook": "start-storybook --static-dir ./target,./node_modules/@jenkins-cd/design-language/dist/assets/,./node_modules/@jenkins-cd/blueocean-core-js/dist/assets/",
```
Comment (michaelneale) [inline blueocean-dashboard/package.json]: typo - should be `../` not `,./` right?

Comment (michaelneale) [inline blueocean-dashboard/package.json]: oh no, it's a path, ignore me.

**Comment by sophistifunk** [comment]
Scrolling issue should be fixed now

**Comment by michaelneale** [comment]
Nearly got all the ATH running. Making it more reliable in the process too.

File: blueocean-dashboard/src/main/js/components/Pipelines.jsx [hunk 2]
```diff
@@ -99,6 +99,35 @@ export class Pipelines extends Component {
                                 );
                             })
                             }
+                            <tr><td>Making shit bigger</td></tr>
```
Comment (michaelneale) [inline blueocean-dashboard/src/main/js/components/Pipelines.jsx]: uh... ?? is this meant to be here?

Comment (sophistifunk) [inline blueocean-dashboard/src/main/js/components/Pipelines.jsx]: No, no it is not.

File: blueocean-dashboard/src/main/js/components/Pipelines.jsx [hunk 3]
```diff
@@ -99,6 +99,35 @@ export class Pipelines extends Component {
...
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
+                            <tr><td>Making shit bigger</td></tr>
```
Comment (michaelneale) [inline blueocean-dashboard/src/main/js/components/Pipelines.jsx]: it still certainly do that, but not sure it was intended to be left in

**Comment by michaelneale** [comment]
passes ATH - looking good. 

Last one for this would be: https://issues.jenkins-ci.org/browse/JENKINS-41547

File: blueocean-core-js/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.57-jm-c",
+  "version": "0.0.57-jm-d",
```
Comment (michaelneale) [inline blueocean-core-js/package.json]: is this the published or unpublished one? (I dont' really know how all this works)

Comment (sophistifunk) [inline blueocean-core-js/package.json]: Published but won't turn up if you ask for "latest". But the version in the branch is the version the branch depends on :)

**Comment by michaelneale** [comment]
When this is merged: https://github.com/jenkinsci/blueocean-acceptance-test/pull/109 needs to be squash and merged at the same time.

**Comment by michaelneale** [comment]
@sophistifunk ATH in better shape, if you can update it again, should be good to go. Let's get it in this morning.

**Comment by michaelneale** [comment]
THIS PASSES GO GO GO GO 🐝

**Comment by i386** [comment]
Tried this out and it is officially HOT 🔥  😍

**Comment by sophistifunk** [comment]
SUCKERS!

*flees*

**Pull request merged**